API-8379 add info about new reports (#911)
livechat/livechat-public-docs: content/data-reporting/changelog/index.mdx

## [v3.4] - Developer preview

### Chats

> - There's a new method, [**Ratings**](/data-reporting/reports-api/v3.4/#ratings), which shows how many chats were rated and how they have been rated during specified period.

Reviewer: ```suggestion
- There's a new method, [**Ratings**](/data-reporting/reports-api/v3.4/#ratings), which shows the number of rated chats along with their ratings during a specified period of time.
```

> - There's a new method, [**Ranking**](/data-reporting/reports-api/v3.4/#ranking), which shows the ratio of good to bad ratings for each operator durning specified period.

Reviewer: ```suggestion
- There's a new method, [**Ranking**](/data-reporting/reports-api/v3.4/#ranking), which shows the ratio of good to bad ratings for each agent within a license.
```
